packages/common/src/store/pages/chat/slice.ts

```diff
@@ -288,6 +323,7 @@ const slice = createSlice({
       // Set the true state
       const { chatId } = action.payload
       delete state.optimisticChatRead[chatId]
+      delete state.optimisticUnreadMessagesCount
```

Comment:
this action needs to update the unreadMessageCount.

the failure case doesn't, so that it goes back to normal